src/applications/disability-benefits/2346/containers/IntroductionPage.jsx

```diff
@@ -30,26 +30,26 @@ class IntroductionPage extends React.Component {
         <div className="process schemaform-process">
           <ol>
             <li className="process-step list-one">
-              <h3>Prepare</h3>
-              <p>To place an order, you'll need your:</p>
+              <h5>Prepare</h5>
```

Comment:
Did this get checked with the axe extension? It looks like a jump from an `h2` to `h5` here, so I'm guessing the extension would yell at you about this.